e2e-tests/IIoTPlatform-E2E-Tests/Properties/launchSettings.json

```diff
@@ -1,9 +1,22 @@
 {
```

Comment:
@dacolgit, please revert all changes to `launchSettings.json` and remove `e2etesting20211202184` resource group as all of its secrets are compromised now. Please be careful not to commit those environment variables in `launchSettings.json` files in future.

Reply:
removed